ui/helpers/http.js

```diff
@@ -43,7 +43,9 @@ export const getData = async function getData(url) {
     failureStatus = response.status;
   }
 
-  if (response.headers.get('content-type') === 'text/html' && failureStatus) {
+  const responseHeader = response.headers.get('content-type').split(';');
```

Comment:
I think naming the variable `contentType` might be clearer?

I was also curious as to whether the `text/html` was always the first element after splitting by semi-colon - and it appears the spec says it is:
https://tools.ietf.org/html/rfc7231#section-3.1.1.1

We could also use `.startsWith()` to avoid the need to `.split()`?

Reply:
Sure, will do.